script.js

```diff
@@ -169,6 +162,7 @@ function addMarker(latLng, id) {
 function setMapOnAll(map) {
   for (var i = 0; i < markers.length; i++) {
     markers[i].setMap(map);
+    console.log("Hello");
```

Comment:
should we start getting rid of these console.logs?

Reply:
Yea definitely. Ill start taking them out as I merge